src/content/docs/tutorial-errors/analyze-your-errors.mdx

```diff
@@ -0,0 +1,124 @@
+
+## Objectives [#objectives]
+
+This doc builds off the problem scoped from the [Scope the problem](/docs/tutorial-errors/scope-the-problem) doc of this tutorial series. In this doc, you will:
+
+* Find the error code associated with a failing API with logs
+* Use stack traces to identify faulty code logic or mistype code  
+
+## Find the source of an error with logs and stack traces [#analyze]
+
+The errors summary tells the story behind your error groups. Depending on how you've configured New Relic, each service might display different kinds of information about an error. For example, disabling distributed tracing limits trace details about external services. Neglecting to set up a log forwarder can similarly limit what attributes your logs show you. For the full errors analysis experience, we recommend these capabilities:
+
+* Enabling distributed tracing if you have it disabled. This is enabled by default.
+* Integrating a log forwarder with New Relic to expand your logs in context experience
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Find the failure point with logs
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Pinpoint errors in your code with stack traces
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+
+    <Steps>
+    <Step>
+    ### Open the Log details window
+
+Looking at our example, our log forwarder has sent full log details to New Relic. This allows us to read denser descriptions about the nature of our error:
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmLogs}
+    />
+
+Our logs in context capability formats your logs information, though you have the option to investigate the unformatted log as well. 
+    </Step>
+    <Step>
+
+    ### Read your logs
+        
+Depending on the nature of the log, you may have more or less logging detail to sort through. Because you spent time upfront narrowing down the service closest to the failure and choosing the likely error group, you have time to read your logs. 
+
+<img
+    title="Overview errors affecting your services"
+    alt="A screenshot showing an app with many errors"
+    src={apmLogsZoom}
+    />    
+
+According to your logs, your problem is:
+
+* A timeout error: `error.error.code: ETIMEDOUT`
+* Related to your customer API: `error.endpoint: customers-api-internal`
+* Agnostic to the kind of request sent to the customer API: `error.request: /api/customers/search/Kirlin/-/`
+
+</Step>
+<Step>
+
+### Find the failure point in your system
+
+Having read your logs, you can now tie it all together. You conclude that there's a dependency failure: All calls to the customer API are timing out and causing upstream requests from `api-gateway` to fail. 
```

Comment:
"having read your logs" is redundant. maybe make this step 'add a message' to the error itself?